### PR TITLE
ddl: add admin batch scan limit

### PR DESCRIPTION
**Pull request by crazycs520**
This PR is for schrodinger-test.
Set admin batch scan size.

### CODE REVIEW

**Comment by crazycs520** [comment]
@winkyao PTAL

File: kv/kv.go [hunk 1]
```diff
@@ -85,6 +85,8 @@ type Retriever interface {
 	// The Iterator must be Closed after use.
 	Seek(k Key) (Iterator, error)
 
+	SeekWithBatchSize(k Key, batchSize int) (Iterator, error)
```
Comment (winkyao) [inline kv/kv.go]: You can add a interface 
```
type Seeker interface{
    SeekWithBatchSize(k Key, batchSize int) (Iterator, error)
}```, then just implement it in tikvSnapshot is ok.

Comment (crazycs520) [reply kv/kv.go]: wonderful idea~

File: store/tikv/snapshot.go [hunk 2]
```diff
@@ -286,6 +286,12 @@ func (s *tikvSnapshot) Seek(k kv.Key) (kv.Iterator, error) {
 	return scanner, errors.Trace(err)
 }
 
+// SeekWithBatchSize implements the Retriever interface.
```
Comment (shenli) [inline store/tikv/snapshot.go]: s/Retriever/kv.Seeker/

File: kv/kv.go [hunk 3]
```diff
@@ -84,13 +84,17 @@ type Retriever interface {
...
-
 	// SeekReverse creates a reversed Iterator positioned on the first entry which key is less than k.
 	// The returned iterator will iterate from greater key to smaller key.
 	// If k is nil, the returned iterator will be positioned at the last key.
 	SeekReverse(k Key) (Iterator, error)
 }
 
+// Seeker is for test.
+type Seeker interface {
+	SeekWithBatchSize(k Key, batchSize int) (Iterator, error)
```
Comment (shenli) [inline kv/kv.go]: Add comments.

**Pull request closed without merging**